users.yml

```diff
@@ -7,17 +7,30 @@
 
   tasks:
     - block:
+        - name: Get list of installed servers
```

Comment:
Let's move to the native module. Something like:
```yaml
- name: Get list of installed servers
  find:
    paths: configs/
    depth: 2
    recurse: true
    hidden: true
    patterns: ".config.yml"
  register: configs
```

Reply:
I'd thought the `find` module only accepted absolute paths.